notification_configuration.go

```diff
@@ -93,12 +92,27 @@ type NotificationConfiguration struct {
 
 // DeliveryResponse represents a notification configuration delivery response.
 type DeliveryResponse struct {
-	Body       string      `json:"body"`
-	Code       int         `json:"code"`
```

Comment:
This never worked as intended. The result would be a `0`. See https://play.golang.org/p/u6bGq49tHa7